frontend/e2e/unauthorizedUser/blocked-features.spec.ts

```
@@ -1,5 +1,13 @@
 import { test, expect } from "@playwright/test";
 
-test.describe("Unauthorized Navigation Tests", () => {});
+test.describe("Unauthorized Navigation Tests", () => {
+    test.use({
+        storageState: "e2e/unauthorizedUser/auth.json",
+    });
+
+    test("Navigate to Dashboard", async ({page}) => {
```

ESLint

[error] 'page' is defined but never used.

(@typescript-eslint/no-unused-vars)